main.py

```diff
@@ -59,7 +59,9 @@ def convert():
     try:
       return perform_notebook_conversion(json)
     except Exception as e:
-      return render_template('jupyter-error.html', error=e.__class__.__name__ + ": " + str(e) + ". ") ,400
+      errMessage = e.__class__.__name__ + ": " + ''.join(str(e).splitlines()) + "."
```

Comment:
Nit, it could be nicer to use an f string here instead of `+` but really not a blocker

Reply:
was gonna suggest that as well!